src/main/java/com/sforce/cd/apexUnit/client/testEngine/AsyncBulkApiHandler.java

```diff
@@ -114,9 +114,6 @@ public String handleBulkApiFlow(PartnerConnection conn, BulkConnection bulkConne
 			ApexUnitUtils
 					.shutDownWithDebugLog(e, "Caught IO exception while trying to deal with bulk connection: "
 							+ e.getMessage());
-		} catch (ConnectionException e) {
```

Comment:
why were you handling this exception before and who is the new successor to handle a broader exception other than above two?